model-optimizer/mo/utils/ir_reader/layer_to_class.py

```diff
@@ -125,8 +125,10 @@ def restore_correct_ports(graph: Graph):
         if 'out' in d:
             node = Node(graph, u)
             num_of_in_nodes = len(node.in_nodes())
-            # we need to check operation type, if it is const op, we don't renumber out ports
-            decremented_number = d['out'] - num_of_in_nodes if node.type != 'Const' else d['out']
+            decremented_number = d['out'] - num_of_in_nodes
+            # we need to check operation type, if it is const op with out_port == 0, we don't renumber out ports
```

Comment:
The comment is useless. It does not explain the logic below. It should say something like this: "Initially Const operation in IR has output port with number 1. But later the behaviour was changed so the output port become 0. This change was made to be consistent with the IR serializer in the IE which generates Const with output port 0. But for the backward compatibility reason for the POT tool the Const output port number should not be changed during reading the IR."

Reply:
Comment updated.